packages/ui/src/components/overlay-container/overlay-container.tsx

```diff
@@ -0,0 +1,56 @@
+import {Component, h, ComponentChild} from 'preact';
+import {ContribLogger, getContribLogger} from '@playkit-js-contrib/common';
+import * as styles from './_overlay-container.scss';
+
+export interface OverlayContainerProps {
+  label: string;
+  content: () => OverlayItemOptions | null;
+}
+
+export interface OverlayItemOptions {
+  label: string;
+  renderContent: () => ComponentChild;
+}
+
+export class OverlayContainer extends Component<OverlayContainerProps> {
+  private _logger: ContribLogger = getContribLogger({
+    module: 'contrib-ui',
+    class: 'OverlayContainer',
+  });
+
+  componentDidMount(): void {
```

Comment:
@semarche-kaltura 
If we do not need this function I do not see a point of applying it with just a log.